canine/localization/file_handlers.py

```diff
@@ -376,13 +395,14 @@ def localization_command(self, dest):
 ## GDC HTTPS URLs {{{
 class HandleGDCHTTPURL(FileType):
     localization_mode = "url"
+    gdc_drs_root = "drs://dg.4dfc:"
```

Comment:
Will DRS URLs always have this root?

Reply:
GDC ones will, yes, that is the one for NCI CRDC.